src/api/Models/WorkflowChange.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.Json.Serialization;
+
+namespace BDMS.Models;
+
+/// <summary>
+/// Represents a status change of a <see cref="WorkflowV2"/>.
+/// </summary>
+[Table("workflow_change")]
+public class WorkflowChange : IIdentifyable, IChangeTracking
+{
+    /// <inheritdoc />
+    [Key]
+    [Column("workflow_change_id")]
+    public int Id { get; set; }
+
+    /// <summary>
+    /// Gets or sets the comment for the change.
+    /// </summary>
+    [Column("comment")]
+    public string Comment { get; set; }
+
+    /// <summary>
+    /// Gets or sets the status of the <see cref="WorkflowV2"/> before the change.
+    /// </summary>
+    [Column("from_status")]
+    public WorkflowStatus FromStatus { get; set; }
+
+    /// <summary>
+    /// Gets or sets the status of the <see cref="WorkflowV2"/> after the change.
+    /// </summary>
+    [Column("to_status")]
+    public WorkflowStatus ToStatus { get; set; }
+
+    /// <summary>
+    /// Gets or sets the id of the <see cref="Workflow"/>.
```

Comment:
Q: Müsste das hier auf `WorkflowV2` verweisen?
Wenn ja entsprechend auch Property umbenennen in `WorkflowV2Id`